libs/testing/docs/index.rst

```diff
@@ -0,0 +1,13 @@
+..
+   Copyright (c) 2019 The STE||AR-Group
+
+   Distributed under the Boost Software License, Version 1.0. (See accompanying
+   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+.. _libs_testing:
+
+=======
+testing
+=======
+
+The testing module contains useful macros for testing like `HPX_TEST` for example.
```

Comment:
`HPX_TEST` should actually be surrounded by double backticks. My bad. @aurianer would you mind updating that?